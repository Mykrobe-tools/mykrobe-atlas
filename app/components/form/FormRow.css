.formRow {
  padding-bottom: 24px;
}
