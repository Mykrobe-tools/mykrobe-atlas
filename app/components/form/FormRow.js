/* @flow */

import React, { Component, PropTypes } from 'react';

import styles from './FormRow.css';

class FormRow extends Component {

  constructor(props: Object) {
    super(props);
  }

  render() {
    const {children} = this.props;
    return (
      <div className={styles.formRow}>
        {children}
      </div>
    );
  }
}

FormRow.propTypes = {
  children: PropTypes.node.isRequired
}

export default FormRow;
