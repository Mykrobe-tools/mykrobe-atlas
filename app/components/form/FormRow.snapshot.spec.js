import React from 'react';
import renderer from 'react-test-renderer';
import FormRow from './FormRow';

describe('FormRow component snapshot', () => {
  const children = '<a href="#">Dolor Sit Amet</a>';

  it('renders correctly', () => {
    const component = renderer.create(
      <FormRow children={children} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
