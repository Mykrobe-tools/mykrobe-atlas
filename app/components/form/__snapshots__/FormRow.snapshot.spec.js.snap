exports[`FormRow component snapshot renders correctly 1`] = `
<div
  className="formRow">
  <a href=\"#\">Dolor Sit Amet</a>
</div>
`;
